src/components/BookstoreInput.jsx

```diff
@@ -0,0 +1,5 @@
+const BookstoreInput = () => (
+  <div>input field here...</div>
+);
+
+export default BookstoreInput;
```

Comment:
I suggest that you use a ```form``` element with inputs for the title and the author of the book as well as a submit button instead of a ```div```. This is because the page would be requiring a user to make some inputs and using a form would be appropriate to handle those inputs. 👍

Reply:
add Form Input